util/util.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"strconv"
+	"strings"
+)
+
+// Difference returns the difference between two arrays
+func Difference(arr1, arr2 []int64) (result []int64) {
+	hash := make(map[int64]bool)
+
+	for _, item := range arr1 {
+		hash[item] = true
```

Comment:
Hashing of an "object" - any considerations that should be taken care of?

Reply:
What do you mean?